src/content/pages/pyladies.md

```diff
@@ -19,60 +19,56 @@ attendees an exclusive opportunity to connect, share, and grow together.
 Join us for a special lunch event aimed at fostering community and empowerment
 in tech. Enjoy meaningful conversations and networking opportunities.
 
-* WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
-* WHERE? Prague Congress Centre (PCC), VIP lunch area.
+- WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
+- WHERE? Prague Congress Centre (PCC), VIP lunch area.
 
 <div class="text-center mb-8">
 <ButtonLink href="https://forms.gle/ts9ryAQj8ieh36NB7">Register for the PyLadies Lunch</ButtonLink>
 </div>
 
-
 ## PyLadies Open Space
 
-During this one-hour PyLadies Open Space session, we invite Pyladies
-organizers, members, newcomers, and allies to join a thoughtful and engaging
-dialogue.
+During this one-hour PyLadies Open Space session, we invite Pyladies organizers,
+members, newcomers, and allies to join a thoughtful and engaging dialogue.
 
 The aim of this session is to discuss the current status of our individual
 PyLadies communities, identify shared challenges, and explore the potential
 benefits of increased collaboration between different PyLadies groups.
 
-Through open and inclusive conversations, we hope to build stronger
-connections, foster mutual growth, and create opportunities for collective
-impact within our diverse PyLadies network.
-
-* WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
-* WHERE? Prague Congress Centre (PCC), VIP lunch area.
+Through open and inclusive conversations, we hope to build stronger connections,
+foster mutual growth, and create opportunities for collective impact within our
+diverse PyLadies network.
 
+- WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
+- WHERE? Prague Congress Centre (PCC), VIP lunch area.
 
 ## #IAmRemarkable
 
```

Comment:
Would be nice to add the official logo. What do you think?
![1_3gH5iAedifINYcm7ioDWcg-513424502](https://github.com/user-attachments/assets/43a057fa-71f4-41f1-8c5f-23cb1bd9b43b)